packages/ui/src/message-room-details-dialog/MessageRoomDetailsDialog.jsx

```diff
@@ -58,6 +61,7 @@ function MessageRoomDetailsDialog({
   const toggleLeaveChannelModal = () =>
     setShowLeaveChannelModal(!showLeaveChannelModal);
   const toggleArchiveChannel = () => setShowArchiveChannel(!showArchiveChannel);
+  const togglePrivateChannel = () => setShowPrivateChannel(!showPrivateChannel);
```

Comment:
do this instead. setShowArchiveChannel(prev => !prev), same for setShowPrivateChannel.

Reply:
done